packages/iframe-execution-environment-service/src/IframeExecutionEnvironmentService.ts

```diff
@@ -266,12 +275,6 @@ export class IframeExecutionEnvironmentService
     );
 
     const commandStream = mux.createStream(PLUGIN_STREAM_NAMES.COMMAND);
-    commandStream.write({
-      jsonrpc: '2.0',
-      method: 'handshake',
-      params: [],
-      id: 0,
-    });
```

Comment:
this looks like leftover from testing, calling write without caring about the result

Reply:
Yeah. We still call the handshake method and check that it succeeds to establish a connection with the worker though, right?